newapi/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ooni-api (1.0.18) unstable; urgency=medium
+
+  * Add database query retry
```

Comment:
I don't see where the query retry is implemented in the provided diff

Reply:
That was related to the original change.

Reply:
Right, so the changelog should change, no?

Reply:
Updated